recipes/freeimage/all/patches/001_disable_TIFF.patch

```diff
@@ -0,0 +1,22 @@
+--- Source/FreeImage/Plugin.cpp
++++ Source/FreeImage/Plugin.cpp
+@@ -254,7 +254,9 @@
+ 			s_plugins->AddNode(InitPNM, NULL, "PPMRAW", "Portable Pixelmap (RAW)", "ppm", "^P6");
+ 			s_plugins->AddNode(InitRAS);
+ 			s_plugins->AddNode(InitTARGA);
++#if 0
+ 			s_plugins->AddNode(InitTIFF);
```

Comment:
Isn't our libtiff recipe working?

Reply:
FreeImage need a private copy of libtiff.

Reply:
Too many modifications? Dammit.

Reply:
It's painful. Look at [vcpkg port](https://github.com/microsoft/vcpkg/tree/master/ports/freeimage), It's really annoying

Reply:
See https://sourceforge.net/p/freeimage/discussion/36111/thread/c326f1be79/
So it's unlikely that something will be done to fix this.

Reply:
I don't really know how to fix it:
- Disabling some features and patching sources
or
- Using embedded deps sources

Both have huge drawbacks.

Reply:
Okay, if you're saying that it's a lot of work to make it truly use external dependencies.

Reply:
I think that big patches to unvendor all these deps -like in the vcpkg port- are fine. The last version of freeimage is 2.5 years old.